python/cuml/svm/svm.pyx

```diff
@@ -138,9 +139,20 @@ class SVC(Base):
         tol : float (default = 1e-3)
             Tolerance for stopping criterion.
         cache_size : float (default = 200 MiB)
-            Size of the kernel cache n MiB
+            Size of the kernel cache during training in MiB. The default is a
```

Comment:
we don't use cache_size param during training, do we? atleast the `fit` method doesn't seem to pass this to libcuml++

Reply:
It is passed through the svmParameter struct.